packages/react-components/src/components/HorizontalGallery.tsx

```diff
@@ -107,6 +107,7 @@ const HorizontalGalleryNavigationButton = (props: {
   const theme = useTheme();
   return (
     <DefaultButton
+      tabIndex={2}
```

Comment:
`tabindex` is tricky as this is a global property and not something that is hierarchical. So if we set tabIndex to one, if Contoso (note: Contoso is our name for another company using our library) embeds our composite into their web page we steal the tabindex order that they may have set e.g.:
```
<body>
<button tabindex={1} />
<button tabindex={2} />
<button tabindex={3} />
<CallComposite />
<button tabindex={4} />
```
The tab index in that order won't work as Contoso expects.

Instead we added a `autofocus` property to the chat composite to allow contoso to say that the composite should take focus when this component is rendered: https://github.com/Azure/communication-ui-library/pull/1235/files#diff-ec0dc325bdf92a107d64cbdd0904c5aae39b00aac3236fe2bfe0b88ee332471b

With the call composite its more complicated as each page of the composite may want to have a different component take auto focus when loaded. This would be a good thing to bring up for discussion on the Web UI teams channel or in standup

Reply:
Removed tab index and leaving autofocus on Microphone